src/main.jsx

```diff
@@ -93,10 +106,22 @@ document.addEventListener('DOMContentLoaded', () => {
     lang: data.cozyLocale
   })
 
+  let history = hashHistory
+  try {
+    var PiwikReactRouter = require('piwik-react-router')
+    const piwikTracker = (Piwik.getTracker(), PiwikReactRouter({
+      url: __PIWIK_TRACKER_URL__,
+      siteId: __PIWIK_SITEID__,
+      injectScript: false
+    }))
+    piwikTracker.push(['enableHeartBeatTimer'])
+    history = piwikTracker.connectToHistory(hashHistory)
+  } catch (err) {}
```

Comment:
As it is for tracking, I silent this intentionally because I don't know what we could do if an error occurs.
If you want to debug this thing, I guess you use a debugger, not a `console.log` debugging workflow. But in production, no need to display something in case of activated DNT option, or other possible cases.